### PR TITLE
Various small fixes, incl. HorizontalScrollable

### PR DESCRIPTION
**Pull request by hisahi**
### Short description
Small various fixes, including adding HorizontalScrollable to other tables and fixing #553.

### DoD
I have:
- [x] added actual code.
- [x] produced clean code that passes ESLint.
- [x] code that actually does what it should.
- [ ] documented the code or added documentation to the wiki.
- [x] added or modified test(s).
- [x] test(s) that actually pass.

### CODE REVIEW

File: labtool2.0/src/components/pages/Courses.js [hunk 1]
```diff
@@ -17,51 +18,56 @@ export class Courses extends Component {
...
-                        )}
-                      </div>
-                    </Table.Cell>
-                    <Table.Cell>{instance.shorterId} </Table.Cell>
-                    <Table.Cell>
-                      <strong>
-                        <Link to={`/labtool/courses/${instance.ohid}`}>{instance.name}</Link>
-                      </strong>
-                    </Table.Cell>
+                <Table.Body>
```
Comment (mjaakko) [inline labtool2.0/src/components/pages/Courses.js]: This file seems to have some spacing issues

File: labtool2.0/src/components/pages/Notification.js [hunk 2]
```diff
@@ -25,13 +25,7 @@ export class Notification extends React.Component {
...
     if (message === undefined) {
       return (
-        <Grid>
-          <Grid.Row>
-            <br />
-            <br />
-            <br />
-          </Grid.Row>
-        </Grid>
+        <div />
```
Comment (mjaakko) [inline labtool2.0/src/components/pages/Notification.js]: Any reason to return `<div />` instead of null?

Comment (hisahi) [reply labtool2.0/src/components/pages/Notification.js]: It adds a bit of a gap where the notification would be, which makes the UI look somewhat better.

File: labtool2.0/src/components/pages/Courses.js (hunk 1 above)
Comment (hisahi) [reply labtool2.0/src/components/pages/Courses.js]: The spacing passes ESLint's prettify as-is

Comment (mjaakko) [reply labtool2.0/src/components/pages/Courses.js]: Might be a problem with GitHub then

File: labtool2.0/src/components/pages/Notification.js (hunk 2 above)
Comment (mjaakko) [reply labtool2.0/src/components/pages/Notification.js]: There would probably be a better way to do it, but let's think about it later

**Comment by gotonode** [comment]
Nice. This source branch was not removed. Feature branches can safely be removed once merged.

**Comment by hisahi** [comment]
The dev branch could be safely removed, as it can be recreated from trunk at any time, but it's better to just not remove it at all.

**Pull request merged**